### PR TITLE
Minor Changes in the Hindi Translation article

### PR DESCRIPTION
**Pull request by Dreadlord075**
* Minor changes from left out English sentences or letters in the Hindi Translation Article.
* The changes are made in the articles_ folder containing How-to-contribute.md, starting-a-project.md and getting-paid.md
* These changes will help in better understanding for Indian users who prefer articles in Hindi

- [✔] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/HEAD/CONTRIBUTING.md)?
- [✔ ] Have you explained what your changes do, and why they add value to the Guides?

### CODE REVIEW

**Review by ahpook — APPROVED** [review]
Thank you for the improvements!

**Comment by DSeerprophetic** [comment]
> Thank you for the improvements!

Google hacking me to the dark web

File: _articles/hi/getting-paid.md [hunk 1]
```diff
@@ -94,14 +94,14 @@ OSS प्रौद्योगिकी उद्योग को बड़े
 
 अपनी व्यक्तिगत परिस्थितियों के आधार पर, आप अपने ओपन सोर्स कार्य के वित्तपोषण के लिए स्वतंत्र रूप से धन जुटाने का प्रयास कर सकते हैं। उदाहरण के लिए:
 
-* @Homebrew (और [many other maintainers and organizations](https://github.com/sponsors/community)) [GitHub Sponsors](https://github.com/sponsors) के माध्यम से उनके काम को वित्तपोषित करें
+* @Homebrew (और [कई अन्य अनुरक्षक और संगठन](https://github.com/sponsors/community)) [GitHub Sponsors](https://github.com/sponsors) के माध्यम से उनके काम को वित्तपोषित करें
```
Comment (Mohsenbahmen) [inline _articles/hi/getting-paid.md:97]: [bugreport-sweet_global2-TKQ1.221114.001-2024-04-06-11-41-50-dumpstate_log-22350.txt](https://github.com/github/opensource.guide/files/14901441/bugreport-sweet_global2-TKQ1.221114.001-2024-04-06-11-41-50-dumpstate_log-22350.txt)

**Review by beans82 — COMMENTED** [review]
b01b93428930f44a78f387b9add183b93dc6c770

**Pull request merged**